packages/framework-integration-tests/integration/provider-unaware/load/scripts/create-products-same-sku.yml

```diff
@@ -0,0 +1,16 @@
+config:
+  target: <not_set>
```

Comment:
This is the artillery script used in one of the tests. As you can see there are many variables with the value `<not_set>`. The value itself doesn't matter, it is just a marker to know which of those variables will be overridden by the typescript code of the integration test.

Reply:
Note: the `target` is always overridden by the ArtilleryExecutor class in a provider unaware way (it just takes the `graphqlURL` of the provider under test)